lib/matplotlib/stackplot.py

```diff
@@ -21,15 +21,15 @@ def stackplot(axes, x, *args,
 
     Parameters
     ----------
-    x : 1d array of dimension N
+    x : array-like (N)
 
-    y : 2d array (dimension MxN), or sequence of 1d arrays (each dimension 1xN)
+    y : sequence of array-like (N) or 2D array (M, N)
```

Comment:
The lack of a comma and swap of order makes this ambiguous as to whether it's "(sequence of 1D) or 2D", or "sequence of (1D or 2D)".

Reply:
```suggestion
    y :  2D array (M, N) or sequence of array-like (N)
```

I agree with @QuLogic about the 'or' here being unclear.

Reply:
and it is made extra awkward because y here may or may not be variadic...